app/models/user.rb

```diff
@@ -5,4 +5,9 @@ class User < ApplicationRecord
     validates :email, presence: true, length: { maximum: 255 }, format: {with: VALID_EMAIL_REGEX}, uniqueness: true
     has_secure_password
     validates :password, presence: true, length: { minimum: 6 }
+
+    def User.digest(string)
```

Comment:
クラスメソッドの定義は`self.hoge`が良さそう

```suggestion
    def self.digest(string)
```